tfhe/src/integer/gpu/mod.rs

```diff
@@ -2248,6 +2253,133 @@ pub unsafe fn unchecked_scalar_left_shift_integer_radix_kb_assign_async<
     update_noise_degree(input, &cuda_ffi_radix_lwe_left);
 }
 
+#[allow(clippy::too_many_arguments)]
+/// # Safety
+///
+/// - [CudaStreams::synchronize] __must__ be called after this function as soon as synchronization
+///   is required
+pub unsafe fn unchecked_scalar_mul_high_async<T: UnsignedInteger, B: Numeric, Scalar>(
```

Comment:
This function should be named `unchecked_scalar_mul_high_integer_radix_kb_async` instead.